tensorly/decomposition/tests/test_candecomp_parafac.py

```diff
@@ -31,6 +31,17 @@ def test_parafac():
     assert_(T.max(T.abs(rec_svd - tensor)) < tol_max_abs,
             'abs norm of reconstruction error higher than tol')
 
+    # Test fixing mode 0 or 1 with given init
+    fixed_tensor = random_kruskal((3, 4, 2), rank=2)
+    rec_svd_fixed_mode_0 = parafac(tensor, rank=2, n_iter_max=2, init=fixed_tensor, fixed_modes=[0])
+    rec_svd_fixed_mode_1 = parafac(tensor, rank=2, n_iter_max=2, init=fixed_tensor, fixed_modes=[1])
+    # Check if modified after 2 iterations
+    tol_fixed = 1e-16
+    assert_(T.norm(rec_svd_fixed_mode_0.factors[0] - fixed_tensor.factors[0]) < tol_fixed,
+            'Fixed mode 0 was modified in candecomp_parafac')
+    assert_(T.norm(rec_svd_fixed_mode_1.factors[1] - fixed_tensor.factors[1]) < tol_fixed,
```

Comment:
```suggestion
   assert_array_equal(rec_svd_fixed_mode_1.factors[1], fixed_tensor.factors[1]) 
```
We can directly use `tensorly.testing.assert_array_equal` for checking that the factors were left untouched.

Reply:
Ok let me correct that quickly